lib/rules/accesskeys.json

```diff
@@ -2,7 +2,7 @@
 	"id": "accesskeys",
 	"selector": "[accesskey]",
 	"excludeHidden": false,
-	"tags": ["best-practice", "cat.keyboard"],
+	"tags": ["cat.keyboard", "best-practice"],
```

Comment:
I swapped the positioning here to better match the order of the other rules' tags.